Stub out a DataStore class and TableView on the History VC

aranasaurus opened the pull request.

Commit:
Add TableView to HistoryViewController

Commit:
Make a DataStore class and use it to load the datas.

Also added some stubbed methods for adding/removing items.

mbcharbonneau commented on Relax/Relax/HistoryItem.swift:
```diff
@@ -0,0 +1,16 @@
+
+import Foundation
+
+struct HistoryItem {
```
Should we add stubs for the NSCoding protocol here?

aranasaurus replied:
I think that'll be better left for an exercise for the workshoppers. Mostly because I can't stub it out without breaking the build.

mbcharbonneau commented on Relax/Relax/DataStore.swift:
```diff
@@ -0,0 +1,59 @@
+
+import Foundation
+
+class DataStore {
```
Another thing that might be useful is the concept of a "current activity"-- a way for the app to know the activity that's been started if the app is closed and re-launched before the user saves it.

Commit:
Add some more stubs as per review comments.

Also added some `assert(false, "Unimplented")`s to the unimplemented bits to make them easier to find.

mbcharbonneau commented:
:+1:
